.github/actions/run-cypress-tests/action.yml

```diff
@@ -80,13 +80,13 @@ runs:
       run: |
         cd ./OpenSearch-Dashboards
         echo "Start checking OpenSearch Dashboards."
-        for i in {1..60}; do
+        for i in {1..90}; do
```

Comment:
qq, why the new magic number? Is there a documentation around what number should be used?

Reply:
No number. Separated out to compile time step. We should get an answer from core on what has changed. Before it was around ~300